Java/PrimeChecker.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;

```

Comment:
Just update the Langauge folder readme, code is fine :-)

Reply:
done bro 👍